vividus-plugin-web-app/src/main/java/org/vividus/selenium/BrowserWindowSizeProvider.java

```diff
@@ -62,21 +63,22 @@ private void checkDesiredBrowserWindowSize(BrowserWindowSize desiredBrowserWindo
         }
     }
 
-    private BrowserWindowSize getScreenSize(boolean remoteExecution)
+    @Override
+    public BrowserWindowSize getScreenSize(boolean remoteExecution)
```

Comment:
there is some ambiguity here: the method name is `getScreenSize` but the result type is `BrowserWindowSize` and class name is `BrowserWindowSizeProvider` (that might be ok for internal `private` method, but not for `public` one)